holdingsiq.json

```diff
@@ -1444,6 +1447,17 @@
                     ],
                     "customDisplayName": "string"
                   }
+                },
+                "bodyoptionn": {
```

Comment:
extra n in bodyoption

Reply:
it comes after bodyoptionm, but it does look like a typo